src/ddmd/statement.d

```diff
@@ -994,6 +998,111 @@ extern (C++) final class ScopeStatement : Statement
 }
 
 /***********************************************************
+ * Scope that contains foreach index variables in a local scope and forwards other members.
```

Comment:
They are in the symbol table. Does this make more sense?:
"Scope whose symbol table contains foreach index variables"